src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,50 @@
 package core.basesyntax;
 
-class Pair {
+import java.util.Objects;
 
+class Pair<T, S> {
+    private T firstItem;
+    private S secondItem;
+
+    public Pair(T firstItem, S secondItem) {
```

Comment:
please check your task according to checklist:
- constructor
https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_06_2.html